src/google-drive/drive.js

```diff
@@ -0,0 +1,162 @@
+let google = require('googleapis');
+let authentication = require("./authentication");
+
+async function createSheet(auth, assignmentName) {
+  var sheets = google.sheets('v4')
+  let response = await sheets.spreadsheets.create({
+    auth: auth,
+    resource: {
+        properties:{
+            title: assignmentName
+        }
+    }
+  })
+  console.log(response.spreadsheetId)
+  return response.spreadsheetId
+}
+
+
+module.exports = {
+
+  
+
+  formatSpreadSheet: function(auth) {
+    var sheets = google.sheets('v4');
+    var spreadsheets_id = "1Kd46uZ3v1WEnhzQbbgDCqGKt9nmPkM-QwZlM1qWb9sw";
```

Comment:
Not sure where this constant is coming from but we'll probably need to read from the database here.